packages/giselle-engine/src/react/generations/generation-runner.tsx

```diff
@@ -110,12 +110,16 @@ function CompletionRunner({
 			return;
 		}
 		addStopHandler(generation.id, stop);
+		const telemetrySettings = {
+			isEnabled: true,
+			metadata: telemetry?.metadata ?? {},
+		};
 		append(
 			{ role: "user", content: "hello" },
 			{
 				body: {
 					generation,
-					telemetry,
+					telemetry: telemetrySettings,
```

Comment:
No need to modify since telemetry is setting here
https://github.com/giselles-ai/giselle/blob/cf5a16cb71fa9be16874aab07993a9ee3190834c/apps/studio.giselles.ai/app/workspaces/[workspaceId]/layout.tsx#L64-L72

Reply:
fixed: 3bca30d8